src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -340,6 +341,38 @@ applyLocal logger dbEnv gasModel pd spv cmdIn mc =
 
       checkTooBigTx gas0 gasLimit (applyPayload em) return
 
+readInitModules
+    :: Logger
+      -- ^ Pact logger
+    -> PactDbEnv p
+      -- ^ Pact db environment
+    -> PublicData
+      -- ^ Contains block height, time, prev hash + metadata
+    -> IO ModuleCache
+readInitModules logger dbEnv pd =
+    evalTransactionM tenv txst go
+  where
+    rk = RequestKey chash
+    nid = Nothing
+    chash = pactInitialHash
+    tenv = TransactionEnv Local dbEnv logger pd noSPVSupport nid 0.0
+           rk 0 permissiveExecutionConfig
+    txst = TransactionState mempty mempty 0 Nothing (_geGasModel freeGasEnv)
+    interp = defaultInterpreter
+
+    go :: TransactionM p ModuleCache
+    go = do
+      readExec <-
+        liftIO $ buildExecParsedCode Nothing
+        "coin.MINIMUM_PRECISION ns.GUARD_SUCCESS"
```

Comment:
ha. Clever. This should only summon `coin` and `ns` yeah?

Reply:
I don't know if it summons `fungible-v2` as Emmanuel's test didn't cover that

Reply:
But yes, just `coin` and `ns`.